app/javascript/components/search/controls/download/DownloadButton.js

```diff
@@ -56,7 +70,26 @@ export default function DownloadButton({ searchResults={} }) {
         </button>
       </OverlayTrigger>
       { showModal &&
-        <DownloadSelectionModal show={showModal} setShow={setShowModal} studyAccessions={matchingAccessions}/> }
+        <DownloadSelectionModal
+          show={showModal}
+          setShow={setShowModal}
+          tdrFileInfo={tdrFileInfo}
+          studyAccessions={matchingAccessions}/> }
     </>
   )
 }
+
+/** limit the files returned to 10, and add two fake analysis files
+  */
+function makeDemoAppropriate(fileInfomation) {
```

Comment:
Same thought here, wondering if something like `makeFilesDemoAppropriate` would be more precise ?

Reply:
Agreed, the variable and the function ideally should align somehow to be more intuitive.